chemprop/models/model.py

```diff
@@ -196,11 +196,12 @@ def predict_step(self, batch: TrainingBatch, batch_idx: int, dataloader_idx: int
         Tensor
             a tensor of varying shape depending on the task type:
 
-            * regression/binary classification: ``n x (t * s)``, where ``n`` is the number of input
-            molecules/reactions, ``t`` is the number of tasks, and ``s`` is the number of targets
-            per task. The final dimension is flattened, so that the targets for each task are
-            grouped. I.e., the first ``t`` elements are the first target for each task, the second
-            ``t`` elements the second target, etc.
+            * | regression/binary classification: ``n x (t * s)``, where ``n`` is the number of input
+              | molecules/reactions, ``t`` is the number of tasks, and ``s`` is the number of targets
+              | per task. The final dimension is flattened, so that the targets for each task are
+              | grouped. I.e., the first ``t`` elements are the first target for each task, the second
+              | ``t`` elements the second target, etc.
+
```

Comment:
I know we just approved this, but I think the "|" weren't necessary (the space addition after the paragraph, however, is.)